core/server/master/src/test/java/alluxio/master/AlluxioMasterRestServiceHandlerTest.java

```diff
@@ -116,7 +117,7 @@ public void before() throws Exception {
     mClock = new ManualClock();
     mExecutorService =
         Executors.newFixedThreadPool(2, ThreadFactoryUtils.build("TestBlockMaster-%d", true));
-    mBlockMaster = new BlockMaster(registry, factory, mClock,
+    mBlockMaster = new BlockMasterFactory().create(registry, factory, mClock,
```

Comment:
this test can use the simpler constructor (`mClock` and `mExecutorService` is not used for anything)